index.html

```diff
@@ -289,6 +289,36 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="#" class="form" method="get">
```

Comment:
- [ ] :wave: The form looks good :+1: but we missed a few details :pinching_hand:  form action value should be similar to ``` https://formspree.io/f/{form_id} ``` and `method` value should equals `POST` I suggest you follow the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) :v: